wunderio/client-UK-FSA-beta
# FSA-925: Consultation filters now standardised like the others

To test:
- go to `/news-alerts/search/consultations`
- the search filter options should be toggleable, functional and look similar to the others
